tika-bundle/pom.xml

```diff
@@ -55,23 +55,21 @@
             <groupId>org.apache.tika</groupId>
             <artifactId>tika-parsers</artifactId>
             <version>${tika.version}</version>
-            <scope>provided</scope>
```

Comment:
📘 Updating the maven bundle plugin required this to be changed to compiled in order to work the same way